miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditStudyController.java

```diff
@@ -105,8 +106,33 @@ public Project populateProject(@PathVariable Long projectId) throws IOException
   public Collection<StudyType> populateStudyTypes() throws IOException {
     return studyService.listTypes();
   }
+  
+  @ModelAttribute("projects")
+  public Collection<Project> populateProjects() throws IOException {
+    return projectService.listAllProjects();
+  }
+
+  // put list of projects into Model object
+  @GetMapping(value = "/new")
+  public ModelAndView newStudy(ModelMap model) throws IOException {
+    User user = securityManager.getUserByLoginName(SecurityContextHolder.getContext().getAuthentication().getName());
+    Study study = new StudyImpl(user);
+    // Project project; // get Project obj from dropdown
+    //
+    // if (Arrays.asList(user.getRoles()).contains("ROLE_TECH")) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

Reply:
@djcooke what should be the approach for resolving the equivalent code at 64 and 141?